kiwi/iso_tools/cdrtools.py

```diff
@@ -250,7 +250,8 @@ def _create_sortfile(self):
             boot_files = list(os.walk(self.source_dir + '/' + self.boot_path))
             boot_files += list(os.walk(self.source_dir + '/EFI'))
             for basedir, dirnames, filenames in boot_files:
-                for filename in filenames:
+                # dirnames.sort() # TODO - seems to break mocking
```

Comment:
Thanks for your effort, looks all good to me except for the TODO here. We don't leave todo's in the kiwi code base, because they usually stay for ever :-) So I looked at that code and I think the sorting problem is also caused by the two lists dirnames and filenames handled separately. So they can be sorted on its own but when the result is written it still can lead to unsorted output in the sort file. I therefore propose the patch in #939 

Feel free to incorporate that one here or let me know if we should merge (after review) #939 first

Thanks